test/api_integration/fixtures/es_archiver/index_patterns/conflicts/mappings.json

```diff
@@ -1,10 +1,10 @@
 {
   "type": "index",
   "value": {
-    "index": "logs-2017.01.01",
+    "index": "logs-2017.01.02",
     "settings": {
       "index": {
-        "number_of_shards": "1",
+        "number_of_shards": "5",
```

Comment:
No reason, no. I didn't manually change that so I'm not sure what happened, but reverted back to 1